include/avif/avif.h

```diff
@@ -176,6 +176,21 @@ typedef enum avifResult
 
 AVIF_API const char * avifResultToString(avifResult result);
 
+// ---------------------------------------------------------------------------
+// avifEncoderHeaderFormat
+
+typedef enum avifEncoderHeaderFormat
+{
+    // Encodes as "avif" brand with a MetaBox and all its required boxes for maximum compatibility.
+    AVIF_ENCODER_FULL_HEADER,
+#if defined(AVIF_ENABLE_EXPERIMENTAL_AVIR)
+    // Encodes as "avir" brand with a CondensedImageBox to reduce the encoded file size.
+    // This is based on the m64572 "Condensed image item" MPEG proposal for HEIF.
+    // WARNING: Experimental feature. Produces files that are incompatible with older decoders.
+    AVIF_ENCODER_REDUCED_HEADER,
```

Comment:
Nit/optional: Consider moving "FULL" and "REDUCED" to the end of the enum names, so that both names start with "AVIF_ENCODER_HEADER". This is a common naming convention. For example,

```
typedef enum avifRange
{
    AVIF_RANGE_LIMITED = 0,
    AVIF_RANGE_FULL = 1
} avifRange;
```